e2etests/web/regular_integration_tests/README.md

```diff
@@ -5,12 +5,16 @@ configuration (e.g. PWA vs non-PWA packaging), please create another
 directory under e2etests/web. Otherwise tests such as text_editing, history,
 scrolling, pointer events... should all go under this package.
 
+Tests can be run on both 'release' and 'profile' modes. However 'release' mode
+will swallow the stack trace and only fails the test. Use 'profile' mode for
```

Comment:
Is it actually swallowing the stack trace or is the stack trace obfuscated?

Reply:
I might have used the wrong word.

It only shows a small message, it does not show the detailed stack trace.

I traced it all the way here: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/foundation/assertions.dart#L501

Reply:
Ah, I'd say "summarize" or "shorten" or something that does not imply that it swallows exceptions, which usually means catch the exception and not print any message at all.

Reply:
Sure, I'll change the wording :)